service/security/shiro/src/test/java/org/eclipse/kapua/service/authentication/shiro/AccessTokenCredentialsImplTest.java

```diff
@@ -17,20 +17,9 @@
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
-import java.lang.reflect.Constructor;
-import java.lang.reflect.Modifier;
-
 @Category(JUnitTests.class)
 public class AccessTokenCredentialsImplTest extends Assert {
 
-    @Test
```

Comment:
Hi @Coduz , just an idea, I think we should add a unit test for the clone constructor too.

Reply:
Done in c7a1322ef12730464753497a60fb9d7a62075d4a